channels/uk.m3u

```diff
@@ -1,4 +1,6 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-country="UK" tvg-language="English" tvg-logo="https://d126oek9ze5bb6.cloudfront.net/uploads/2018/07/4Music_primary.png" group-title="Music",4Music (576p) [not 24/7]
+http://csm-e.tm.yospace.com/csm/extlive/boxplus01,4music-desktop.m3u8?yo.up=http://boxtv-origin-elb.cds1.yospace.com/uploads/4music/
```

Comment:
This link can be simplified : http://csm-e.tm.yospace.com/csm/extlive/boxplus01,4music-desktop.m3u8